src/azul/transformer.py

```diff
@@ -454,8 +454,11 @@ def accumulate(self, value) -> bool:
         else:
             return False
 
-    def get(self) -> List[Any]:
-        return list(self.value)
+    def get(self, sort: bool = True) -> List[Any]:
```

Comment:
Can you explain why this needs to be modal? Why can't it always sort?

Reply:
The `get` method of `SetAccumulator` requires a `sort` param because the `get` method of the child class `SetOfDictAccumulator` calls `super().get()` and fails when attempting to produce a sorted list from a set of frozendict.

Reply:
This isn't at all obvious and would have deserved an explanation in the form of a comment.

However, I think there are better ways of dealing with this. I am starting to think that we should move the sorting out of the accumulators and sort at the call sites to get(). We can then either have a protocol by which the caller interrogates the accumulator if the accumulated value is sortable or catch an exception that unmistakably indicates the inability to sort it. Or we just make sure that frozendict is sortable by implementing __lt__. If you do this, there maybe some interaction/overlap with `sort_frozen`.